src/components/atoms/switch/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import clsx from "clsx"
+import * as _Switch from "@radix-ui/react-switch"
```

Comment:
Nit: Rename this to `RadixSwitch` as we have done in other components that Radix components